drivers/scheduler/k8s/k8s.go

```diff
@@ -748,7 +748,8 @@ func (k *K8s) getAddressesForNode(n corev1.Node) []string {
 func (k *K8s) parseK8SNode(n corev1.Node) node.Node {
 	var nodeType node.Type
 	var zone, region string
-	if k8sCore.IsNodeMaster(n) {
+
+	if k8sCore.IsNodeMaster(n) && k.NodeDriverName != "ibm" {
```

Comment:
Should we add this condition in https://github.com/portworx/sched-ops/blob/5a6a43c6a5b38a858b5366cf5ecf4f0f11056f00/k8s/core/nodes.go#L137 since this will be a case for all IBM clusters?